src/main/java/com/lambdaschool/oktafoundation/models/PartnerApplication.java

```diff
@@ -1,6 +1,7 @@
 package com.lambdaschool.oktafoundation.models;
 
 import javax.persistence.*;
+import java.util.Date;
```

Comment:
I'd left a random comment on a different pull request asking that we use java.time.LocalDateTime instead of java.util.Date. Let me see if I can find the information. Found it. I'll just put it here as well:

Random java quirk. `java.util.Date` is no fun at all. I unknowingly tried to use it in a previous project and ran into tons of problems. It worked fine on the backend. But getting a date object from JSON or any sort of string was impossible.

Can we convert this, and anywhere else we may be implementing it, to `java.time.LocalDateTime` instead? That should work just fine with the JS's Date() object.

See [this article](https://programminghints.com/2017/05/still-using-java-util-date-dont/) for more details